.travis.yml

```diff
@@ -6,32 +6,40 @@ env:
-## The iOS 8.1 simulator is commented out because it does not start. It looks a problem of the simulator.
-## Uncomment the following line after the simulator problem is fixed.
-#    - DESTINATION="OS=8.1,name=iPhone 4S" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=9.3,name=iPhone 6" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=10.3.1,name=iPhone 6S Plus" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" PLATFORM="iOS" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=11.2,name=iPhone 8 Plus" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" PLATFORM="iOS" POD_LINT="YES" ACTION="test"
-    - DESTINATION="arch=x86_64" SCHEME="SwinjectStoryboard-OSX" SDK="macosx" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=9.2,name=Apple TV 1080p" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=10.2,name=Apple TV 1080p" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" POD_LINT="NO" ACTION="test"
-    - DESTINATION="OS=12.0,name=Apple TV 4K" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" POD_LINT="NO" ACTION="test"
+    - PROJECT=SwinjectStoryboard.xcodeproj  
+matrix:
+  include:
+    - env: JOB="POD_LINT"
+      osx_image: xcode10
+      script:
+        - bundle exec pod repo update
+        - bundle exec pod lib lint
+    - env: JOB="XCODE" DEST="OS=9.3,name=iPhone 6" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="OS=10.3.1,name=iPhone 7 Plus" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="OS=11.2,name=iPhone 8 Plus" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="OS=12.2,name=iPhone X" SCHEME="SwinjectStoryboard-iOS" SDK="iphonesimulator" ACTION="test"
+      osx_image: xcode10.2
+    - env: JOB="XCODE" DEST="arch=x86_64" SCHEME="SwinjectStoryboard-OSX" SDK="macosx" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="arch=x86_64" SCHEME="SwinjectStoryboard-OSX" SDK="macosx" ACTION="test"
+      osx_image: xcode10.2
+    - env: JOB="XCODE" DEST="OS=9.2,name=Apple TV 1080p" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="OS=10.2,name=Apple TV 1080p" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" ACTION="test"
+      osx_image: xcode10
+    - env: JOB="XCODE" DEST="OS=11.0,name=Apple TV 4K" SCHEME="SwinjectStoryboard-tvOS" SDK="appletvsimulator" ACTION="test"
+      osx_image: xcode10
 before_install:
   - curl -L -O https://github.com/Carthage/Carthage/releases/download/0.28.0/Carthage.pkg
   - sudo installer -pkg Carthage.pkg -target /
   - rm Carthage.pkg
   - carthage bootstrap --platform iOS,macOS,tvOS --cache-builds
```

Comment:
You're using the `--cache-builds` flag, but not requesting Travis to actually cache them.
You can add:
```yaml
cache:
  directories:
    - Carthage
```

Reply:
👍